src/quo2/components/record_audio/record_audio/buttons/record_button.cljs

```diff
@@ -1,7 +1,5 @@
 (ns quo2.components.record-audio.record-audio.buttons.record-button
```

Comment:
nit: if anyone feels like doing some refactoring, these components are in the wrong part of the codebase :) should be aligned with figma file structure convention 👍